webapp/src/actions/index.ts

```diff
@@ -44,13 +56,15 @@ export const openCreateModal = (postId) => {
     };
 };
 
-export const openCreateModalWithoutPost = (description, channelId) => (dispatch) => dispatch({
-    type: ActionTypes.OPEN_CREATE_ISSUE_MODAL_WITHOUT_POST,
-    data: {
-        description,
-        channelId,
-    },
-});
+export const openCreateModalWithoutPost =
+  (description, channelId) => (dispatch) =>
+      dispatch({
+          type: ActionTypes.OPEN_CREATE_ISSUE_MODAL_WITHOUT_POST,
+          data: {
+              description,
+              channelId,
+          },
+      });
```

Comment:
Can we revert all of these styling changes not related to this PR?

Reply:
Only parts of the code related to using the siteURL should be edited in this PR

Reply:
@mickmister  have removed all unwanted styles and checked only the changes to this PR is done .